src/psbt.js

```diff
@@ -49,6 +49,8 @@ import {generateBip32DerivationByIndex, generateBraid} from './braid';
  *
  */
 
+export const PSBT_MAGIC_HEX = "70736274ff";
+export const PSBT_MAGIC_B64 = "cHNidP"; 
```

Comment:
I probably didn't actually compute this back then, but it should be 

`cHNidP8=` as a conversion of 0x70736274FF to base64

Reply:
Yeah, I saw that too but didn't want to rock the boat too much. In fact, I think it needs to be without the `=` though as that's added by base64 as a buffer for the end of a string I think. Anyway, that breaks the tests but adding an `8` at the end to finish the conversion seems to work (and then lower down `substring(0,7)`

Reply:
That's what it looks like from the [test vectors](https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki#test-vectors) too.